storage.bs

```diff
@@ -178,7 +178,18 @@ larger <a>site storage quota</a>. Factors such as navigation frequency, recency
 bookmarking, and <a href="#persistence">permission</a> for {{"persistent-storage"}} can be used as
 indications of "popularity".
 
+The <dfn export>application cache site storage usage</dfn> for an origin is a rough estimate of the amount
+of bytes used in Application Cache in <var>origin</var>'s <a>site storage unit</a>. [[HTML#appcache]]
```

Comment:
I think these should be normative references, `[[!`, not informative ones, `[[`

Reply:
It looks like section links don't work with normative references.  I wanted to use the section link to point specifically at the AppCache part of the HTML spec and Caches part of the Service Worker spec.  Do you think I should prioritize the section link or normative reference?

Reply:
Normative reference. If you want to link to application cache, you can link "Application Cache" there.

Reply:
Could you specify what you mean by "there"?  How would I link to application cache?

Reply:
`<a>Application Cache</a>` should work.